climpred/tests/conftest.py

```diff
@@ -251,6 +276,30 @@ def hindcast_hist_obs_1d(
     return hindcast
 
 
+@pytest.fixture
+def hindcast_recon_1d_mm(hindcast_recon_1d_ym):
```

Comment:
`"""HindcastEnsemble with initialized and reconstruction (observations) as a monthly time series (no grid)."""`